.gitignore

```diff
@@ -22,3 +22,9 @@ Debug
 Debug_Build
 __history
 __astcache
+app/consapp/convbin/gcc/sbp2rinex
```

Comment:
do we need to add these files in?

Reply:
no these are the compiled binaries when you run the GCC makefiles, so we don't want git to check them in